mslice/presenters/workspace_manager_presenter.py

```diff
@@ -127,7 +127,12 @@ def _rename_workspace(self):
             self._workspace_manager_view.error_select_only_one_workspace()
             return
         selected_workspace = selected_workspaces[0]
-        new_name = self._workspace_manager_view.get_workspace_new_name()
+        try:
+            new_name = self._workspace_manager_view.get_workspace_new_name()
```

Comment:
The `try-except` statements could then be replaced by a check to see if `new_name` is `None`, and if it is then return.